amaConnectorCfg.ini

```diff
@@ -0,0 +1,31 @@
+### Config File - This file contains the main settings for the AmaConnector
+
+[MAIN]
+# Path to avalanche directory or where data shall be exported to
+avalancheDir = data/amaExports
+
+# Path to access file relative to AmaConnector
+accessFile = access.txt
+
+# command to perform query of data base
+queryString = select * from event_full
+#where not st_isempty(geom_rel_event_pt)"
+
+# configuration of database
+dBConfiguration = avaframe
+
+# type of events that shall be exported: recorded, design
+eventType = recorded
+
+# constraint to only fetch avalanches that have release event line and event deposition line geometries
+constraint =
+
+# projection
+projstr =
```

Comment:
Either set a default value or at least give examples of what's expected in the commen (True/False/ Some Value)

Reply:
in the orig avaframeexport.py where these are required they are hardcoded to be empty str, so I have put that here too as default, but wanted to have them in the ini so that they can be chosen differently..

Reply:
Yes, but what is e.g. expected in the contraint field? ist that True? ist it False? is it something else?
Same for the projstr? How does it have to look? EPSG:29393? WGS84?